android/libraries/rib-base/src/main/java/com/badoo/ribs/core/routing/configuration/action/single/NoOpAction.kt

```diff
@@ -2,11 +2,31 @@ package com.badoo.ribs.core.routing.configuration.action.single
 
 import android.os.Parcelable
 import com.badoo.ribs.core.routing.configuration.ConfigurationContext.Resolved
+import com.badoo.ribs.core.routing.configuration.ConfigurationKey
 import com.badoo.ribs.core.routing.configuration.action.ActionExecutionParams
+import com.badoo.ribs.core.routing.transition.TransitionElement
 
-internal object NoOpAction : ResolvedSingleConfigurationAction() {
+internal class NoOpAction<C : Parcelable>(
+    item: Resolved<C>
+) : Action<C> {
 
-    override fun <C : Parcelable> execute(item: Resolved<C>, params: ActionExecutionParams<C>): Resolved<C> =
-        // no-op
+    override fun onBeforeTransition() {}
+    override fun onTransition() {}
+    override fun onFinish() {}
+    override fun reverse() {}
+    override val result: Resolved<C> =
         item
+
+    override val transitionElements: List<TransitionElement<C>> =
+        emptyList()
+
+    object Factory :
```

Comment:
why new line here?

Reply:
AS is funny sometimes when moving things between packages. It adds full package name which then exceeds max line length, breaks to new line, then removes package name, but leaves line break.

Fixed.